index.html

```diff
@@ -0,0 +1,150 @@
+                    </div>
+                    <h3> CC Exhibition</h3>
+                    <p> Appreciate various creations by talented people. </p>
+                </li>
+                <li class="program-list-item">
+                    <div class="program-list-item-img">
+                        <img src="./img/program_icon_03.png" alt="">
+                    </div>
+                    <h3> Forum</h3>
+                    <p> Have a chance to share your thoughts with people with like-minded people. </p>
+                </li>
+                <li class="program-list-item">
+                    <div class="program-list-item-img">
+                        <img src="./img/roasted-chicken.png" alt="">
+                    </div>
+                    <h3> Meal </h3>
+                    <p> Give a meal to people in need in Maputo communities. </p>
+                </li>
+                <li class="program-list-item">
+                    <div class="program-list-item-img">
+                        <img src="./img/charity.png" alt="">
+                    </div>
+                    <h3> Donate </h3>
+                    <p> Donate and help improve the lives of people in Maputo communities. </p>
+                </li>
+            </ul>
+            <button class="program-btn"> Join CC Global Summit 2022 </button>
+            <a href="#" class="see-program"> SEE THE WHOLE PROGRAM </a>
+        </section>
+        <section class="featured-speakers">
+            <div class="featured-speakers-title">
+                <h2> Members </h2>
+                <div class="separator separator-features-speakers"></div>
+            </div>
+            <ul class="featured-speakers-list">
+   
+            </ul>
+            <button class="show-more-speakers">
+                <span> More </span>
+                <img src="./img/down-arrow.png" alt="">
+            </button>
+        </section>
+        <section class="footer">
+            <div class="partners">
+                <div class="logo-info-title partners-title">
+                    <h2> Partner </h2>
+                    <div class="separator separator-logo-info"></div>
+                </div>
+                <div class="partner-logo">
+                    <p class="partner-logo-p partner-logo-p-1"> Google </p>
+                    <p class="partner-logo-p partner-logo-p-2"> mozilla </p>
+                    <p class="partner-logo-p partner-logo-p-3"> NAVER </p>
+                    <p class="partner-logo-p partner-logo-p-4"> Pepsi </p>
+                    <p class="partner-logo-p partner-logo-p-5"> EJ Media </p>
+                </div>
+            </div>
+            <div class="rights">
+                <div class="company">
+                    @Projecto Muda
+                </div>
+                <div class="rights-reserved">
+                    2022 Project Muda. <span> All rights reserved</span>
+                </div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to wrap this container into a `footer` tag :wink: 
- [ ] I highly recommend you separate the `footer` from the `partners section` in order to follow the best HTML practices as suggested by the previous reviewer. :wink:

Reply:
- Good job following the advice of the previous reviewer 👏 👏